src/v/json/allocator.h

```diff
@@ -0,0 +1,27 @@
+#include <rapidjson/allocators.h>
+namespace json {
+class throwing_allocator : public rapidjson::CrtAllocator {
```

Comment:
There's now a `CrtAllocator` base, and member (they're stateless, but it's confusing).

I think remove the base and implement `Free`.

Reply:
oh good catch!